greptimedb/results/m6i.8xlarge_bluesky_100m.json

```diff
@@ -0,0 +1,36 @@
+{
+  "system": "GreptimeDB",
+  "version": "v0.13.0-nightly-20250315",
+  "os": "Ubuntu 24.04",
+  "date": "2025-03-17",
+  "machine": "m6i.8xlarge, 10000gib gp3",
+  "cluster_size": 1,
+  "comment": "",
+  "retains_structure": "yes",
+  "tags": [
+  ],
+  "dataset_size": 100000000,
+  "dataset_size_readable": "100m",
+  "num_loaded_documents": 99999968,
+  "data_compression": "auto",
+  "total_size": 10824010671,
+  "total_size_readable": "10.82 GB",
+  "data_size": 10823634560,
+  "data_size_readable": "10.82 GB",
+  "index_size": 348815,
+  "index_size_readable": "349 KB",
+  "result": [
+    [0.496, 0.167, 0.165],
```

Comment:
My local measurements on `m6i.8xlarge`:

```
[0.647, 0.165, 0.166]
[10.563, 3.276, 3.177]
[0.286, 0.222, 0.221]
 [0.121, 0.061, 0.065]
[0.283, 0.204, 0.2]
```

These are sufficiently close to your measurements, thanks!